secondPart/4_convertObject.js

```diff
@@ -0,0 +1,14 @@
+const removeNullandUndefinedValues = (obj) => {
+    const convertedObj = Object.assign({}, obj)
+    Object.keys(convertedObj).forEach(key => {
+        if (convertedObj[key] === null || convertedObj[key] === undefined) {
+            delete convertedObj[key];
```

Comment:
never use delete for removing properties. That's bad practice actually. Create new object depending on your filtered result

Reply:
Ok, I will remember this, thanks.